mysql-test/suite/innodb/t/insert_into_empty.test

```diff
@@ -495,3 +495,16 @@ DROP TABLE t1;
 CREATE TABLE t (a CHAR CHARACTER SET utf8) ENGINE=InnoDB ROW_FORMAT=REDUNDANT;
 INSERT t SELECT left(seq,1) FROM seq_1_to_43691;
 DROP TABLE t;
+
+--echo #
+--echo #  MDEV-32453  Bulk insert fails to apply when trigger
+--echo #                  does insert operation
+--echo #
+SET AUTOCOMMIT=0;
+CREATE TABLE t(c INT)ENGINE=InnoDB;
+CREATE TRIGGER t2_ai AFTER INSERT ON t FOR EACH ROW SET @a:=(SELECT * FROM t);
+INSERT INTO t VALUES (1);
+--error ER_SUBQUERY_NO_1_ROW
+INSERT INTO t VALUES (1);
+DROP TABLE t;
+SET AUTOCOMMIT=1;
```

Comment:
This test is demonstrating the _correct_ operation _after_ the bug has been fixed, and **not** the _incorrect_ operation (→ crash) _before_ the bug has been fixed.

Right?

Perhaps you can clarify this in the description in the commit and PR.

Reply:
Normally, if the code fix is reverted, the regression test is expected to crash or fail. If that is not the case here, it should be clarified in the commit message. I did not check that yet.